dipy/denoise/tests/test_nlmeans.py

```diff
@@ -90,8 +90,15 @@ def test_nlmeans_4d_3dsigma_and_threads():
     sigma = np.ones(data.shape[:3])
     mask = np.zeros(data.shape[:3])
 
-    # mask[25-10:25+10] = 1
-    mask[:] = 1
+    # If you don't provide large enough datasets the OS or OpenMP may not give
+    # all threads to all CPUs
+    # We use False by default to make the test run faster.
+    heavy_duty = False
```

Comment:
Maybe better to do this through a global switch (something like "fast" and using `skipif`)?